rdr_service/genomic/genomic_job_components.py

```diff
@@ -1694,6 +1716,51 @@ def calculate_contamination_category(self, sample_id, raw_contamination, member:
 
         return contamination_category
 
+    def _set_metrics_array_data_file_paths(self, row: dict) -> dict:
+        gc_site_bucket_map = config.getSettingJson(config.GENOMIC_GC_SITE_BUCKET_MAP)
+        site_id = self.file_obj.fileName.split('_')[0].lower()
```

Comment:
Can add a fallback for these configs entries so if they are not present the code doesn't fail?

Reply:
I've added defaults and a conditional return so we won't get a runtime error if a value can't be found in the config.